javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -149,94 +149,204 @@ console.log(theHobbit.info());
+
+#### 3. ...that the original object _inherits_ from, and has access to all of its prototype's methods and properties
+
+Here, the "original object" refers to an object like `player1` or `player2`. These objects are said to "inherit", or simply said, these objects have access to the `prototype`'s properties or functions, if they have been defined. For example, if there was a `.sayHello()` function defined on the `prototype`, `player1` can access the function just as if it was it's own function - `player1.sayHello()`. But it's not just `player1` who can call the `.sayHello()` function, even `player2` can call it, since it's defined on the `prototype`! Read on to know the details of how it works and how you could do this yourself!
+
+#### Accessing an object's prototype
+
+Conceptually, you now might feel like you know, or at least have an idea of what a `prototype` of an object is. But how do you _know_ or actually _see_ what the prototype of an object is? Let's find out. You can try running the following code in the developer console of your browser. (Make sure you've created the `player1` and `player2` objects from before!)
 
 ~~~javascript
-function Student(name, grade) {
-  this.name = name
-  this.grade = grade
-}
+player1.__proto__ === Player.prototype // returns true
+player2.__proto__ === Player.prototype // returns true
+~~~
 
-Student.prototype.sayName = function() {
-  console.log(this.name)
-}
-Student.prototype.goToProm = function() {
-  console.log("Eh.. go to prom?")
+Now, to understand this code, let's use the three points from earlier:
+
+1. **All objects in JavaScript have a `prototype`**:
+
+   1a. You can check the object's `prototype` by using the `.__proto__` property of the object, like `player1.__proto__`.
+
+   1b. This `.__proto__` property refers to the `.prototype` property of the Object Constructor (i.e., `Player(name, marker)`) - `player1.__proto__ === Player.prototype`.
+
+2. **The prototype is another object**: 
+
+   2a. The _value_ of the Object Constructor's `.prototype` property (i.e., `Player.prototype`) contains the `prototype` object. 
+   
+   2b. The _reference_ to this value of `Player.prototype` is stored in every `Player` object's `.__proto__` property, every time a `Player` object is created. 
+   
+   2c. Hence, you get a `true` value returned when you compute `player1.__proto__ === Player.prototype`.
+
+3. **...that the original object _inherits_ from, and has access to all of its prototype's methods and properties**: 
+
+   3a. As said in the earlier point, every `Player` object's `.__proto__` property refers to `Player.prototype`: `player1.__proto__ === player2.__proto__` (returns `true`).
+   
+   3b. So, any properties or methods defined on `Player.prototype` will be available to the created `Player` objects!
```

Comment:
Please refer to the style guide on how we number our markdown [ordered lists (lazy numbering) ](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lazy-numbering)